notebooks/Text_Generation_Training.ipynb

```diff
@@ -17,20 +17,20 @@
       "cell_type": "code",
```

Comment:
I'm unsure if anything besides metadata has changed in this notebook that should be reviewed? If not, we could also revert changes here.

Reply:
In `Text_Generation_training` there was a cell that had an extra indent - so I just un-indented it just as a quick fix, but I realized that i didn't commit those changes in yet, will do so now